Azure/azure-cli
# {Packaging} MSI: Use `WixBroadcastEnvironmentChange` and set `WIXUI_EXITDIALOGOPTIONALTEXT`

**Description**
This is an internal ask from the CLI team to make some minimal changes regarding environment variables.

* Use built-in env var propagation
* Show message to restart terminals first time

**Testing Guide**
1. Install new MSI produced with changes. Verify message shows up on exit dialog.
2. Uninstall MSI.
3. Install older MSI (no message; wasn't added)
4. Install new MSI again. Verify no message because the old version already set the PATH.

Technically, someone could install the old version and turn right around and install the new one without restarting terminals, but that seems unlikely. It's also a point in time problem. Once the "older MSI" in the scenario above has this dialog change, it would've showed the message anyway. Thus, I don't recommend showing this message more often that not. It would be unnecessary for terminals to be restarted (potentially losing command history, session variables, etc.) if the PATH wasn't effectively changed during an upgrade.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
